src/picongpu/include/particles/densityProfiles/LinearExponentialImpl.hpp

```diff
@@ -18,15 +18,14 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
-
 #pragma once
 
 #include "simulation_defines.hpp"
 
+
 namespace picongpu
 {
-
-namespace gasProfiles
+namespace densityProfiles
```

Comment:
OPTIONAL:
In this file several variables are still names `gas...` - might be good to change naming as well